### PR TITLE
add task catalog grid solution

### PR DESCRIPTION
**Pull request by booobryyyk**
[DEMO LINK](https://maksym-bobryk.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/blocks/_catalog.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $card-size);
+  gap: 48px;
+  justify-content: center;
+
+  $media-querry:
+    2 488px,
+    3 768px,
+    4 1024px;
```
Comment (ghost) [inline src/styles/blocks/_catalog.scss:10]: good implementation 🔥 but you also can include your default template to this loop
```suggestion
  display: grid;
  gap: 48px;
  justify-content: center;

  $media-querry:
  	1 0
    2 488px,
    3 768px,
    4 1024px;
```

**Review by ghost — APPROVED** [review]

File: src/styles/blocks/_catalog.scss (hunk 1 above)
Comment (bohdan-mykhailenko) [inline src/styles/blocks/_catalog.scss:10]: Maybe it's better to use maps in the _variables(_maps) file.

**Review by bohdan-mykhailenko — APPROVED** [review]
Great job 👍

File: src/styles/blocks/_catalog.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  @each $number, $wide in $media-querry {
+    @media (min-width: $wide) {
+      grid-template-columns: repeat($number, $card-size);
+    }
+  }
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/_catalog.scss:16]: The use of this loop worsens the readability of the code. In order to fully comprehend what is happening, one must check the $media-query variable.

**Review by Ihor-Karpyn — APPROVED** [review]
GJ, left one suggestion

**Pull request closed without merging**